src/app/core/learning-object-module/file/file.service.ts

```diff
@@ -167,6 +144,36 @@ export class FileService {
     return response;
   }
 
+  /**
+   * Calls LO service to update the packageable status of toggled files
+   *
+   * @param learningObjectID The current learning object's ID
+   * @param fileIDs An array of file IDs that need to be updated
+   * @param state The new packageable property to update to
+   * @returns A promise
+   */
+  // TODO: Move to bundling service
+  toggleBundle(
+    learningObjectId: string,
+    fileIDs: string[],
+    state: boolean
+  ) {
+    return this.http
+      .patch(
+        FILE_MANAGER_ROUTES.UPDATE_BUNDLE(learningObjectId),
+        {
+          fileIDs: fileIDs,
+          packagable: state
+        },
+        { headers: this.headers, withCredentials: true }
+      )
+      .pipe(
+
+        catchError(this.handleError)
+      )
+      .toPromise();
+  }
+
```

Comment:
will this be moved in joaquin's PR?

Reply:
No, because this is supposed to be in the file service. Clark-service and gateway have this route organized here. The bundling router only has two routes.